app/serializers/question_serializer.rb

```diff
@@ -0,0 +1,43 @@
+# == Schema Information
+#
+# Table name: questions
+#
+#  id                 :integer          not null, primary key
+#  question_type      :integer
+#  building_type_id   :integer
+#  parent_option_type :string
+#  parent_option_id   :integer
+#  category_id        :integer
+#  text               :string
+#  status             :integer
+#  created_at         :datetime         not null
+#  updated_at         :datetime         not null
+#  parameter          :string           not null
+
+class QuestionSerializer < ActiveModel::Serializer
+  attributes :id,
+             :question_type,
+             :options,
+             :building_type_id,
+             :parent_option_type,
+             :parent_option_id,
+             :category_id,
+             :text,
+             :status,
+             :parameter
+
+  has_many :answers
```

Comment:
you might need to remove this since we don't really want every single answer associated with a question to be delivered via json